master/internal/api_trials.go

```diff
@@ -195,14 +197,38 @@ func (a *apiServer) GetExperimentTrials(
 	})
 
 	a.sort(resp.Trials, req.OrderBy, req.SortBy, apiv1.GetExperimentTrialsRequest_SORT_BY_ID)
-	return resp, a.paginate(&resp.Pagination, &resp.Trials, req.Offset, req.Limit)
+	if err := a.paginate(&resp.Pagination, &resp.Trials, req.Offset, req.Limit); err != nil {
+		return nil, err
+	}
+
+	trialIds := make([]string, 0)
+	for _, trial := range resp.Trials {
+		trialIds = append(trialIds, strconv.Itoa(int(trial.Id)))
+	}
+
+	switch err := a.m.db.QueryProto(
+		"proto_get_trials_plus",
+		&resp.Trials,
+		strings.Join(trialIds, ","),
+	); {
```

Comment:
I wonder if they're an easier way to do this. I can't think of anything, but out of curiosity what else have you considered?

Reply:
do you mean for passing params to posgres?

Reply:
if this wasn't an arbitrary list of trials we could SELECT it from the experiment id. I also saw some support for this from postgres driver in go (I have to look that up) but it's not exposed through any of our named-query helpers.

Reply:
This isn't a _lot_ better, but if you wrap the stringified list in `{}`, you should be able to then do `$1::int[]` in the SQL, which feels a little nicer than `string_to_array`.